src/App.tsx

```diff
@@ -1,61 +1,139 @@
 import './App.scss';
+import { useEffect, useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './type/user';
+import { Todo } from './type/todo';
+
+function getUser(userId: number): User | undefined {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || undefined;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [selectedTitle, setTitle] = useState('');
+  const [selectedUser, setUser] = useState('');
+  const [addNewTodo, setTodo] = useState(todos);
```

Comment:
Names are incorrect. Make sure that setState function is named according to the noun variable, e.g. [title, setTitle]
https://react.dev/reference/react/useState